altanalyze3/components/annotation/main.py

```diff
@@ -38,9 +38,9 @@ def __init__(self, host=None, path=None, port=None):
             use_cache (bool): Whether to cache requests.
         """
         # Use defaults if arg is None.
-        host = host or DEFAULT_HOST
-        path = path or DEFAULT_PATH
-        port = port or DEFAULT_PORT
+        host = DEFAULT_HOST if host is None else host
```

Comment:
What if we just use DEFAULT_URL instead of a combination of host, port, and path. Then you can delete a bunch of functions and in the parser allow to set `--url`